08-rails/requests-and-responses.md

```diff
@@ -0,0 +1,82 @@
+# Requests and Responses: HTTP Verbs, CRUD, and Status Codes
+## Learning Goals
+
+- Learn about the different kinds of HTTP requests (the HTTP _verbs_)
+- Discuss the situations for which each _verb_ is suited
+- Explore how the acronym _CRUD_ (<b>C</b>reate <b>R</b>ead <b>U</b>pdate and <b>D</b>elete) relates to both making interactive web applications and HTTP _verbs_.
+- Return a status code from a Rails application
```

Comment:
This indicates that status codes are an important part of this lecture but we actually don't talk about them at all - am i missing something?

Reply:
From the PR:
>Requests and responses lesson (previously HTTP verbs, CRUD and idempotency) needs to be redone. Want to talk about status codes!